app/views/home/landing.html.erb

```diff
@@ -45,10 +45,10 @@
         </div>
       </div>
       <section class="queryOptionsContainer" id="default">
-        <% @capabilities.to_a.sample(5).each do |c| %>
+        <% @capabilities.to_a.each do |c| %>
           <div data-type="capabilities" class="pill white"><%= c.name %></div>
         <% end %>
-        <% @materials.to_a.sample(5).each do |m| %>
+        <% @materials.to_a.each do |m| %>
```

Comment:
Same for materials, we might want to show only some

Reply:
This was a petition made by the client.